### DISCUSSION TITLE
Seriously, Activity McActivityFace? You couldn't stay put for five minutes?

### DISCUSSION THREAD

**Discussion by rubiidev18alt** [cytrinc/bark-coding#75] [General]
Alright, people!  Listen up.  Ugh, here we go again.  Seems like our little friend, the Activity, decided to play escape artist again.  Seriously, did someone leave the containment unit open?  Looks around accusingly  No?  Great.  Rolls eyes

Anyway, for the newbies, here's what to do.  First, don't freak.  Freaking out makes this way worse, trust me.  Second, grab your squirt gun filled with whatever keeps this thing calm (glitter? essential oils? who even knows?).  Third, distract it with something.  Shiny object?  Ball of yarn?  Doesn't matter, just get its attention away from whatever it's about to mess up.  Fourth, and most important, shove that little monster back in its box!  Be creative!  Music?  Lecture?  Doesn't matter, just get it contained.  Look, if we don't work together on this, who knows what kind of disaster we'll be facing next time.  Ugh, this is getting annoying.

**Comment by raynepaws** [comment]
????????

**Reply by rubiidev18alt** [reply]
the activity is gone again basically

**Reply by raynepaws** [reply]
it's been 18 hours since the last commit, what do you mean the activity is gone??
_also, [think before you pin](https://github.com/mariocraft987/bark.github.io/discussions/47)_

**Comment by mariocraft987** [comment]
> it's been 18 hours since the last commit, what do you mean the activity is gone?? _also, [think before you pin](https://github.com/mariocraft987/bark.github.io/discussions/47)_

I agree
